### DISCUSSION TITLE
Bed Visualizer is accessible with basic view permissions via web interface

### DISCUSSION THREAD

**Discussion by Scutz** [jneilliii/OctoPrint-BedLevelVisualizer#580] [General]
The Bed Visualizer Plugin is being shown with basic permissions for accounts that login via the webinterface. 

I tried creating an account that only has the following rights: "Settings Access, Status, Webcam" to watch my printer via work. With these permissions it was possible to access the "Bed Visualizer" Tab and start a mesh update. The damage possible is not really high, but it would be better from the privace perspective to block this setting for accounts with these permissions. Can you update this?

Thank you for your nice plugin :)

**Comment by jneilliii** [comment]
could you share a screenshot of the user permissions as you set them?

**Comment by Scutz** [comment]
Sure, it is german but it shows what I mean. The tabs Temperature and Steuerung (Control) are accessible but you can not access the controls in them. Only the "Update Mesh Now" button is accessible.
![grafik](https://user-images.githubusercontent.com/59345048/209400194-7254ec2e-7ac0-476d-b4d8-81cdef26ede1.png)
and
![grafik](https://user-images.githubusercontent.com/59345048/209400212-371a1c1d-5fd1-413c-b937-77ed54d2659a.png)
and the access possibility
![grafik](https://user-images.githubusercontent.com/59345048/209400531-877e3b41-7d66-4bb4-9389-e2a549786897.png)
